pkg/reconciler/ingress/ingress_test.go

```diff
@@ -1261,6 +1264,60 @@ func TestReconcileProbing(t *testing.T) {
 func makeLoadBalancerNotReady(i *v1alpha1.Ingress) {
 	i.Status.MarkLoadBalancerNotReady()
 }
+func TestReconcileProbingOffClusterGateway(t *testing.T) {
+	table := TableTest{
+		{
```

Comment:
formating
```suggestion
	table := TableTest{{
```

Take a look at the other tests we usually fold the start of the array `{` with the first array entry bracket `{`

Reply:
oh yeah sorry, I forgot to put that back.

for whatever reason, my eyes have a harder time keeping track of which pairs of curly braces go together in the usual format lol